src/components/Status/Status.module.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../styles/colors';
+
+.Status {
+  &[data-status-type='active'] {
+    --zui-status-background: #{$green-9};
+  }
+
+  &[data-status-type='idle'] {
+    --zui-status-background: #{$orange-10};
+  }
+
+  &[data-status-type='busy'] {
+    --zui-status-background: #{$read-10};
+  }
+
+  &[data-status-type='offline'] {
+    --zui-status-background: #{$grey-10};
+  }
+
+  &[data-status-type='unread'] {
+    --zui-status-background: #{$purple-11};
+  }
+}
+
+.Status {
+  display: flex;
+  width: 8px;
+  height: 8px;
+  border-radius: 9999px;
+  background: var(--zui-status-background);
+}
```

Comment:
I like the structure of this file, but I think we could simplify and make it a little more readable. If your up for it could you try something like the following and let me know what you think?

I've changed the order slightly and removed the var(). Also I don't think the `#{}` is required for the colours...I could be wrong though. Could you also try replacing `border-radius: 9999px;` with `50%` for the same outcome?

```
@import '../../styles/colors';

.Status {
	display: flex;
	width: 8px;
	height: 8px;
	border-radius: 50%;

	&[data-variant='active'] {
		background: $green-9;
	}

	&[data-variant='idle'] {
		background: $orange-10;
	}

	&[data-variant='busy'] {
		background: $read-10;
	}

	&[data-variant='offline'] {
		background: $grey-10;
	}

	&[data-variant='unread'] {
		background: $purple-11;
	}
}

```

Reply:
Changed the border radius to %. Changed size from px to rems. As per brett's comments If there is a variation of styles for the same element we are declaring the variables accordingly ([link](https://github.com/zer0-os/zUI/pull/72#discussion_r1038658400) ).  If we are using imported colors inside the variable we use `#{}`.

Reply:
Looks good. I'll aim to look through this again over the weekend. Thanks for making those other changes !

Reply:
Looking good to me, just one small request - rather than declaring `.Status {}` twice could we nest all styles into one? Very minor, but doing so will improve readability, I haven't run this myself but this should have the same outcome 

```
.Status { 
   display: flex; 
   width: 8px; 
   height: 8px; 
   border-radius: 50%
   
   &[data-variant='active'] {
     background: $green-9; 
    } 
    
    &[data-variant='idle'] {
      background: $orange-10; 
     }
     
     ....
```